### PR TITLE
Add black to neurom, format everything, and add to lint

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -35,8 +37,10 @@ commands =
 basepython=python3.7
```
Comment (adrien-berchet) [inline tox.ini:37]: You could add `skip_install = true` in this section.

**Comment by mgeplf** [comment]
GIthub turned on handling of `.git-blame-ignore-revs` (https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/), so we'll add the commit there, once it has a stable hash

**Review by mgeplf — APPROVED** [review]

**Review by mgeplf — CHANGES_REQUESTED** [review]
Let's allow single quotes w/ `--skip-string-normalization`, and rerun the format.

**Comment by eleftherioszisis** [comment]
@mgeplf is it good now?

**Review by mgeplf — APPROVED** [review]

**Pull request merged**